### PR TITLE
Revert "Update index.html"

### PR DESCRIPTION
**Pull request by johaan1515**
Reverts CodeForMyFriends/create-a-simple-web-the-cool-gang#5

### CODE REVIEW

**Comment by johaan1515** [comment]
why

**Comment by Star-Cyber3** [comment]
no

File: index.html [hunk 1]
```diff
@@ -3,4 +3,3 @@
   <h1 poopoo boi is not cool>
```
Comment (Star-Cyber3) [inline index.html:3]:
```suggestion
  <h1>poopoo boi is not cool</h1>
```

**Review by Star-Cyber3 — CHANGES_REQUESTED** [review]
Wrong code

**Comment by Star-Cyber3** [comment]
Remember to adjust the code

**Pull request closed without merging**